mass_editing/models/mass_editing_line.py

```diff
@@ -1,51 +1,67 @@
 # Copyright (C) 2019 - Today: GRAP (http://www.grap.coop)
+# Copyright (C) 2020 Iván Todorovich (https://twitter.com/ivantodorovich)
 # @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from odoo import api, fields, models
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
 
 
 class MassEditingLine(models.Model):
     _name = "mass.editing.line"
-    _order = "sequence,field_id"
     _description = "Mass Editing Line"
+    _order = "sequence, field_id"
 
     sequence = fields.Integer()
-
-    mass_editing_id = fields.Many2one(comodel_name="mass.editing")
-
+    server_action_id = fields.Many2one(
+        "ir.actions.server",
+        string="Server Action",
+        ondelete="cascade",
+        required=True,
+    )
     model_id = fields.Many2one(
-        comodel_name="ir.model", related="mass_editing_id.model_id"
+        "ir.model",
+        related="server_action_id.model_id",
     )
-
     field_id = fields.Many2one(
-        comodel_name="ir.model.fields",
+        "ir.model.fields",
         string="Field",
-        domain="["
-        "('name', '!=', '__last_update'),"
-        "('ttype', 'not in', ['reference', 'function']),"
-        "('model_id', '=', model_id)"
-        "]",
+        domain="""
+            [
+                ("name", "!=", "__last_update"),
```

Comment:
You should include here any of the "magic fields". There's a variable in Odoo core (in upper case) that contains the list of all of them

Reply:
Actually they're not the same.
I think the developer intention was to ignore only this field: https://github.com/odoo/odoo/blob/14.0/odoo/models.py#L380
Magic fields like `create_date`, `write_date`, `create_uid`, `write_uid` can be mass editted, I don't see any reason not to..

I did do this: https://github.com/OCA/server-ux/pull/255/commits/5b63806003e8215e780a2a171e736b8fe59403fa, because `id` shouldn't be mass editted either

Reply:
That's not true: `create_date` and so on are automatically removed from any create/write dict.

Reply:
Done!